xbmc/cores/AudioEngine/Utils/AEUtil.cpp

```diff
@@ -554,12 +554,7 @@ AVSampleFormat CAEUtil::GetAVSampleFormat(AEDataFormat format)
     case AEDataFormat::AE_FMT_RAW:
       return AV_SAMPLE_FMT_U8;
     default:
-    {
-      if (AE_IS_PLANAR(format))
```

Comment:
if you argue like this, you need to remove entire default but this is not what we want to achive with the default case.
coverity is wrong here.